logic/src/commonTest/kotlin/com/wire/kalium/logic/data/call/CallRepositoryTest.kt

```diff
@@ -41,7 +45,175 @@ class CallRepositoryTest {
         }
     }
 
+    @Test
+    fun givenEmptyListOfCalls_whenGetAllCallsIsCalled_thenReturnAnEmptyListOfCalls() {
+        val calls = callRepository.getAllCalls()
+
+        assertEquals(0, calls.value.size)
+    }
+
+
+    @Test
+    fun givenAListOfCallProfiles_whenGetAllCallsIsCalled_thenReturnAnEmptyListOfCalls() {
+        callRepository.updateCallProfileFlow(
+            CallProfile(
+                mapOf(
```

Comment:
Could you extract this map so it is defined on the beginning of this test?

This way we can do:

```kotlin

val callsMap = mapOf(...)


assertEquals(callsMap.size, calls.size)
```

And if we change the tests, we don't need to update the values in two places.